plugins/fzf/fzf.plugin.zsh

```diff
@@ -71,11 +71,19 @@ function fzf_setup_using_debian() {
 
   case $PREFIX in
     *com.termux*)
+      if [[ ! -f "${PREFIX}/bin/fzf" ]]; then
+        # fzf not installed
+        return 1
+      fi
       # Support Termux package
       completions="${PREFIX}/share/fzf/completion.zsh"
       key_bindings="${PREFIX}/share/fzf/key-bindings.zsh"
       ;;
     *)
+      if [[ ! -f /usr/bin/fzf ]]; then
```

Comment:
maybe we should just add a check here that `/usr/share/doc/fzf/examples/` exists

Reply:
Hmm, something OOTB would be nice as well, instead of having to set a `skip-dpkg` option.

Reply:
Yup. Checking for `/usr/share/doc/fzf/examples/` is meant to solve this without an extra option.